catalyst_voices/packages/internal/catalyst_voices_view_models/lib/src/proposal/funded_proposal.dart

```diff
@@ -8,7 +9,7 @@ final class FundedProposal extends Equatable {
   final String campaignName;
   final String category;
   final String title;
-  final DateTime fundedDate;
+  final DateTime? fundedDate;
```

Comment:
`FundedProposal` does not have `fundedDate`?

Reply:
The field from where this one is initialized is nullable, I'll add there `!` to make it non-nullable here. Should be not null but who knows what backend will send.